weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/HumanFeedback/humanFeedbackTypes.ts

```diff
@@ -0,0 +1,25 @@
+import {AnnotationSpec} from '../../pages/wfReactInterface/generatedBaseObjectClasses.zod';
+import {Feedback} from '../../pages/wfReactInterface/traceServerClientTypes';
+
+export const HUMAN_FEEDBACK_BASE_TYPE = 'wandb.annotation';
+export const FEEDBACK_TYPE_OPTIONS = ['string', 'number', 'boolean', 'array'];
+export type FeedbackSchemaType = (typeof FEEDBACK_TYPE_OPTIONS)[number];
+
+export const makeAnnotationFeedbackType = (type: string) =>
+  `${HUMAN_FEEDBACK_BASE_TYPE}.${type}`;
+
+export type tsHumanAnnotationSpec = AnnotationSpec & {
+  ref: string;
+};
+
+export type HumanAnnotationPayload = {
+  value: {
+    [key: string]: {
```

Comment:
is value always a dict always ? Can't it just be a primitive?

Reply:
Feedback payloads are always of the type: 
```
value: {
   [annotationSpecName]: {
      [artifactVersion]: value
   }
}         
```

where the second "value" can be anything, including primatives